### DISCUSSION TITLE
做文本检测训练时ppocr INFO: cur metric, precision: 0, recall: 0.0, hmean: 0

### DISCUSSION THREAD

**Discussion by jingmingtao** [PaddlePaddle/PaddleOCR#12845] [Q&A]
这样的情况，我定位到可能是因为我的数据长宽、大小问题不支持吗，只是支持大图片，小字吗。我都是用paddleocr进行标注，外阔了小图，变成大图后，就好了。但是小图大字时，就会有。我无法解决，希望大佬可以看到给一点建议，修改哪个地方阿，官方爸爸

**Comment by zhangqiqi1228** [comment]
> 这样的情况，我定位到可能是因为我的数据长宽、大小问题不支持吗，只是支持大图片，小字吗。我都是用paddleocr进行标注，外阔了小图，变成大图后，就好了。但是小图大字时，就会有。我无法解决，希望大佬可以看到给一点建议，修改哪个地方阿，官方爸爸

你好，我也遇到了相同的问题，请问你找到解决方案了吗？

**Reply by tea0403** [reply]
你好，请问您后来解决这个问题了吗？
